backup.sh

```
@@ -2,7 +2,7 @@
 << help
```

Shellcheck (0.10.0)

[error] Couldn't parse this here document. Fix to allow more checks.

(SC1073)